app/src/main/java/org/schulcloud/mobile/data/model/CurrentUser.java

```diff
@@ -15,6 +24,8 @@
     public String schoolId;
     public String displayName;
 
+    public RealmList<RealmString> permissions;
+
     public String get_id() {
```

Comment:
Well the id is naming `_id` as default in our MongoDB-Models. We don't want to convert any property name after fetching it from the server so we don't rename the `_id` either.